### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by danilodileo**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/metatdenovo/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/metatdenovo _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

I updated the documentation based on issues #281  and #273

### CODE REVIEW

**Comment by erikrikarddaniel** [comment]
This is against master, that's not your intention, right?

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 ## Introduction
 
-**nf-core/metatdenovo** is a bioinformatics best-practice analysis pipeline for assembly and annotation of metatranscriptomic data, both prokaryotic and eukaryotic.
+**nf-core/metatdenovo** is a bioinformatics best-practice analysis pipeline for assembly and annotation of metatranscriptomic and metagenomic data, both prokaryotic, eukaryotic and viruses.
```
Comment (erikrikarddaniel) [inline README.md]:
```suggestion
**nf-core/metatdenovo** is a bioinformatics best-practice analysis pipeline for assembly and annotation of metatranscriptomic and metagenomic data from prokaryotes, eukaryotes or viruses.
```

File: docs/usage.md [hunk 2]
```diff
@@ -185,6 +173,40 @@ A more targeted annotation option offered by the workflow is the possibility for
 Each HMM file will be used to search the amino acid sequences of the ORF set and the results will be summarized in a tab separated file in which each
 ORF-HMM combination will be ranked according to score and E-value.
 
+#### How to download manually the databases for functional annotation
```
Comment (erikrikarddaniel) [inline docs/usage.md]:
```suggestion
#### How to manually download the databases for functional annotation
```

**Review by erikrikarddaniel — APPROVED** [review]
Two small suggestions, otherwise :+1:

**Comment by danilodileo** [comment]
> This is against master, that's not your intention, right?

I changed it!

**Pull request merged**